src/PKSim.Core/CoreConstants.cs

```diff
@@ -286,7 +286,8 @@ public static class CalculationMethod
          {
             LINKS_COMMON, 
             SPECIFIC_CLEARANCE, 
-            DISTRIBUTION_IN_VITRO_PKSIM
+            DISTRIBUTION_IN_VITRO_PKSIM,
+            DYNAMIC_SUM_FORMULAS
```

Comment:
Otherwise the new reaction parameters were not created.
Needed quite a while to figure it out :)